konghao/server/src/server.cpp

```diff
@@ -0,0 +1,32 @@
+//
+// Created by andykong on 22-6-23.
+//
+#include "ros/ros.h"
+#include "std_msgs/Header.h"
+#include <client/Num.h>
+#include <client/Print.h>
+
+void chatterCallback(const client::print::Request &req,
+                     const client::print::Response &res)
+{
+    ros::NodeHandle n;
+    ROS_INFO("name:%s", req.name.c_str());
+    res.result = "OK";
+    ros::Subscriber client_topic_pub = n.subscribe<std_msgs::Header>("head", 10, show_data);
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]